backend/apps/owasp/management/commands/owasp_sync_board_members.py

```
@@ -0,0 +1,178 @@
+            "entity_type": entity_type,
+            "member_name": member_name,
+            "role": EntityMember.Role.MEMBER,
+            "is_active": True,
+        }
+
+        matched_user = self._fuzzy_match_github_user(member_name)
+        if matched_user:
+            data["member"] = matched_user
+
+        try:
+            entity_member = EntityMember.update_data(data, save=True)
```

Ruff (0.13.2)

Local variable `entity_member` is assigned to but never used

Remove assignment to unused variable `entity_member`

(F841)